client/src/app/site/pages/meetings/services/meeting-settings-definition.service/meeting-settings-definitions.ts

```diff
@@ -651,6 +651,18 @@ export const meetingSettings: SettingsGroup[] = [
                     }
                 ]
             },
+            {
+                label: _(`Projector`),
+                settings: [
+                    {
+                        key: `motions_block_slide_columns`,
+                        label: _(`Maximum number of columns on motion block slide`),
+                        type: `integer`,
+                        helpText: _(`Default is 3`),
```

Comment:
Adjust to the configured default if needed.

Reply:
Updated the validator.